modules/payment/commerce_payment.post_update.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for Payment.
+ */
+
+/**
+ * Update user Remote ID field data using payments gateways instead of module.
+ * @see https://www.drupal.org/node/2861181
+ */
+function commerce_payment_post_update_1(&$sandbox) {
+  // Get the user ids that have 'commerce_remote_id' field not emtpy.
+  $result = \Drupal::entityQuery('user')->exists('commerce_remote_id')->execute();
```

Comment:
Wouldn't we only want to run this the first time, and I think we could just do a count query